Python/ceval.c

```diff
@@ -740,14 +743,26 @@ _PyEval_MatchClass(PyThreadState *tstate, PyObject *subject, PyObject *type,
     if (PyObject_IsInstance(subject, type) <= 0) {
         return NULL;
     }
+    // Short circuit if there aren't any arguments:
+    Py_ssize_t nkwargs = PyTuple_GET_SIZE(kwargs);
+    Py_ssize_clean_t nattrs = nargs + nkwargs;
```

Comment:
Don't use `Py_ssize_clean_t` explicitly:
```suggestion
    Py_ssize_t nattrs = nargs + nkwargs;
```